### DISCUSSION TITLE
posts/why_i_am_disappointed_with_the_third_generation_of_wolf/

### DISCUSSION THREAD

**Discussion by giscus** [HalfSweet/Blog#23] [comment]
# posts/why_i_am_disappointed_with_the_third_generation_of_wolf/

最近第三代郊狼开始内测了，作为一名电子爱好者，自然不能错过这个机会。于是我以原价 368 元的主机，以及 128 元两个智能按钮的价格，购买了一套第三代郊狼。 失望是从快递到手开始的，当到手之后我拆开，发现里面的竟然是飞机盒而不是之前的推拉式盒子。这个细节让我感觉有些失望，不过在打开盒子之后有了一些慰藉。因为里面给了一个收纳包，这点很好，当时我甚至很兴...

https://cf.blog.halfsweet.cn/posts/why_i_am_disappointed_with_the_third_generation_of_wolf/

**Comment by KDCJT** [comment]
这个执行标准好像是企标，也算是合规的

**Comment by CaiSong123** [comment]
这是什么东西，一脸懵逼进来，搜半天没搞明白

**Comment by SaltedFishesNG** [comment]
草，一个图片13MB大，为什么不压缩一下喵（

价格方面缺乏竞品是这样的...
其实主要竞品只有役次元，然后役次元影响力不如DG-LAB，以至于郊狼一度成为电击器代名词（仅限中国大陆）

监管这个倒还好，因为法律始终跟不上科技脚步；
现在不少国标都是企标改出来的。
